templates/event/event_show.html.twig

```diff
@@ -106,7 +106,64 @@
                 {% if event.description %}
                     <p class="p-1">{{event.description}}</p>
                 {% endif %}
+                {% if event.isMixDesk() %}
+                    <p class="p-1"> <strong> Эквалайзер: </strong>
+                    <div id="wrapper0">
+                    </div>
+                    <div id="wrapper">
+                    </div>
+                    <div id="wrapper2">
```

Comment:
@daria-sieroshtan 
please give proper names to these
Also update names of JS scripts